UGENE-5730 Exporting to existing and opened document

rasputinkirill opened the pull request.

rasputinkirill committed:
UGENE-5730 Exporting to existing and opened document

yalgaer reviewed (changes requested):
The fix is very different from what is described in the bug.
If a user wants to overwrite the file UGENE should overwrite.

rasputinkirill committed:
Remove not actual test

rasputinkirill commented:
> The fix is very different from what is described in the bug. If a user wants to overwrite the file UGENE should overwrite.

File will be overwritten in current version. I just changed 'error' type of message and removed test which tests corresponding error message. Because that old test case check test behavior that not actual in current version.

yalgaer reviewed (approved):
Thank you for the details. Approved
